tests/dialects/test_vertica.py

```diff
@@ -0,0 +1,88 @@
+from tests.dialects.test_dialect import Validator
+
+
+class TestVertica(Validator):
+    dialect = "vertica"  # type:ignore
+    maxDiff = None  # type:ignore
```

Comment:
Are these `type:ignore` still needed here?

Reply:
mypy error we are getting, we tried with other connectors in sqlglot we got the same error, that's why we added

Reply:
Make sure to format your PR's with sqlglot's `make style`, and for the specific versions of the packages you can run `make install-dev`. Also, the same formatter is ran as part of the ci/cd tests on every commit, so if something breaks you can also see it on this PR.

Let me know if that solution drops the error.